.github/workflows/main.yml

```diff
@@ -22,6 +23,10 @@ jobs:
     - name: Install dependencies
       run: |
         pip install .
+        pip install flake8
+    - name: flake8 linting
+      run: |
+        flake8 $(git ls-files onyo/*.py onyo/commands/*.py tests/*.py)
```

Comment:
`find` might be a better solution here, as it will be recursive, and thus less brittle. Something like:
```
find . -type f -name "*.py"
```